src/nepaliDatePicker.js

```diff
@@ -809,6 +809,22 @@ var calendarFunctions = {};
         daysNumFromMinBsYear += bsDate + 1;
       } else if (bsYear === 2081 && bsMonth === 12) {
         daysNumFromMinBsYear += bsDate - 1;
+      } else if (bsYear === 2082 && bsMonth === 2) {
```

Comment:
This is forcefull way of doing it, instead i think we can update the core data.

Reply:
Exactly. I guess the core maintainers have to take a crack at it. This fix is just a brute force method intended to get the job done for a couple of months. We are bound to face this issue again next year or a year after.